beacon-chain/core/electra/deposits.go

```diff
@@ -524,11 +524,12 @@ func GetValidatorFromDeposit(pubKey []byte, withdrawalCredentials []byte, amount
 	validator := &ethpb.Validator{
 		PublicKey:                  pubKey,
 		WithdrawalCredentials:      withdrawalCredentials,
+		EffectiveBalance:           0,
+		Slashed:                    false,
```

Comment:
Please update the godoc to reflect these changes. Here and in the other change. Thanks

Reply:
@jtraglia - I'm talking about the copy of the spec definition on lines 506-522

Reply:
Got it. Done. Sorry about that.